resources/roadmap/index.md

```diff
@@ -3,145 +3,147 @@ layout: custom
 title: Roadmap
 ---
 <div class="roadmap">
-<div class="text-center pre-roadmap container">
-    <div class="row">
-       <div class="col-xs-4">
-            <li class="completed"></li>
-            <p>Completed task</p>
-       </div>
-       <div class="col-xs-4">
-            <li class="ongoing"></li>
-            <p>Ongoing task</p>
-       </div>
-       <div class="col-xs-4">
-            <li class="upcoming"></li>
-            <p>Upcoming task</p>
-       </div>
+    <div class="text-center pre-roadmap container">
```

Comment:
Hmm, better fix indentation in a dedicated PR. This just makes it nearly impossible to review.